### PR TITLE
add flux config for xui

### PR DESCRIPTION
**Pull request by ernestman28**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -199,6 +199,17 @@ k8s/ithc/common/xui/webapp.yaml @hmcts/xui
 k8s/ithc/common/xui/manage-organisations.yaml @hmcts/xui
 k8s/ithc/common/xui/approve-org.yaml @hmcts/xui
 k8s/ithc/common/xui/share-case.yaml @hmcts/xui
+k8s/perftest/common/xui/xui-tc.yaml @hmcts/xui
+k8s/perftest/common/xui/webapp.yaml @hmcts/xui
+k8s/perftest/common/xui/manage-organisations.yaml @hmcts/xui
+k8s/perftest/common/xui/approve-org.yaml @hmcts/xui
+k8s/perftest/common/xui/share-case.yaml @hmcts/xui
+k8s/demo/common/xui/xui-tc.yaml @hmcts/xui
```
Comment (adusumillipraveen) [inline CODEOWNERS]: Demo doesn't need code owners as of now.

File: k8s/demo/common/xui/xui-tc.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: helm.fluxcd.io/v1
```
Comment (adusumillipraveen) [inline k8s/demo/common/xui/xui-tc.yaml]: Do u know demo doesn't work same as other environments with current setup?

File: CODEOWNERS (hunk 1 above)
Comment (ernestman28) [reply CODEOWNERS]: okay I will remove this

File: k8s/demo/common/xui/xui-tc.yaml (hunk 2 above)
Comment (ernestman28) [reply k8s/demo/common/xui/xui-tc.yaml]: I only know we can use Product chart or Microservice chart in demo, but didnt know the actual setup will be different in flux config. Can you point me in the right direction to setup demo env for using microservice chart? thx

Comment (adusumillipraveen) [reply k8s/demo/common/xui/xui-tc.yaml]: I mean environment itself is setup differently. For example, it doesn't have connectivity to existing postgres databases. so, ur config won't wait.

Comment (ernestman28) [reply k8s/demo/common/xui/xui-tc.yaml]: so what we need to do if we would like to connect to a PaaS postgre instance? i.e  xui-terms-and-conditions-demo . Or the demo environment doesnt support connect to any PaaS DB?

Comment (adusumillipraveen) [reply k8s/demo/common/xui/xui-tc.yaml]: It doesn't support connecting to any Paas DB.

**Pull request merged**